ibis/backends/base/sql/alchemy/query_builder.py

```diff
@@ -262,7 +262,7 @@ def _add_select(self, table_set):
             return result.select_from(table_set)
 
         # otherwise we expect a single FROM in `result`
-        assert num_froms == 1, f"num_froms == {num_froms:d}"
+        # assert num_froms == 1, f"num_froms == {num_froms:d}"
```

Comment:
If I keep this assertion then `ibis/backends/tests/test_aggregation.py::test_filter` is failing for certain SQL backends.

Reply:
Well, if everything else is passing then it's probably fine to remove!

Reply:
Can you delete this line?

Reply:
Actually, this is broken.

Reply:
Ah, I think we need to move the assertion. One sec, I'll fix it up.

Reply:
Thanks!